BlazarUI/app/scripts/components/shared/LazyRender.jsx

```diff
@@ -44,10 +44,6 @@ let LazyRender = React.createClass({
   },
 
   getHeight: function(numChildren, childHeight, maxHeight) {
-    let fullHeight = numChildren * childHeight;
-    if (fullHeight < maxHeight) {
-      return fullHeight;
-    }
     return maxHeight;
   },
 
```

Comment:
I would pass this in as an optional prop, otherwise it limits the use of this component elsewhere.